tools/Datadog.Core.Tools/EnvironmentTools.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.IO;
+using System.Linq;
+
+namespace Datadog.Core.Tools
+{
+    /// <summary>
+    /// General use utility methods for all tests and tools.
+    /// </summary>
+    public class EnvironmentTools
+    {
+        private static string _solutionDirectory = null;
+
+        /// <summary>
+        /// Find the solution directory from anywhere in the hierarchy.
+        /// </summary>
+        public static string GetSolutionDirectory()
```

Comment:
This seems like a better place for helper utilities. I'll expand on it later and convert uses of EnvironmentHelper.